test/shared/setup.ts

```diff
@@ -368,7 +368,7 @@ export async function singleSlotProofSetup(block: any, proofs: any) {
   const minVotingDuration = BigInt(0);
   const maxVotingDuration = BigInt(2000);
   const votingStrategies: bigint[] = [BigInt(singleSlotProofStrategy.address)];
-  const votingStrategyParams: bigint[][] = [[proofInputs.ethAddressFelt, BigInt(1)]];
+  const votingStrategyParams: bigint[][] = [[proofInputs.ethAddressFelt, BigInt(0)]]; // For the aave erc20 contract, the balances mapping has a storage index of 0
```

Comment:
Mind expanding on that? It is common? Is it expected?

Reply:
for the test we are proving aave balances, and in their specific erc20 implementation the slot index of the `balances[]` mapping is 0. Not always o though, which is annoying as each DAO using it will have to work out their own slot index. Ive got a rough tool to do it in the evm-slot-key-verification repo